turbinia/workers/__init__.py

```diff
@@ -121,9 +121,12 @@ def close(self, task, success, status=None):
       return
     self.successful = success
     self.run_time = datetime.now() - self.start_time
-    if not status:
+    if not status and self.successful:
       status = 'Completed successfully in {0:s} on {1:s}'.format(
           str(self.run_time), self.worker_name)
+    elif not status and not self.successful:
+      status = 'Completed unsuccessfully in {0:s} on {1:s}'.format(
```

Comment:
I would suggest changing this text to something less visually similar than `[un]successful`. e.g. `Failed to run in...`